README.md

```diff
@@ -57,3 +57,10 @@ In order to have a fully functional blog, the SQL file contains :
 # Sending emails
 
 Emails are sent with `php mailer` _package_. To send emails, create `mail.yaml` configuration file. An example configuration is available in `mail.yaml.example` file.
+
```

Comment:
Where is composer install

Reply:
The documentation for _composer_ is available on line 9.